Source/UI/NewUI/PaymentForm/View/CardNumber/CardNumberView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CardNumberView.swift
+//  Frames
+//
+//  Created by Harry Brown on 05/07/2022.
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import UIKit
+
+protocol CardNumberViewProtocol: AnyObject {
+  var schemeIcon: Constants.Bundle.SchemeIcon { get set }
+}
+
+final class CardNumberView: UIView, CardNumberViewProtocol {
+  private let viewModel: CardNumberViewModelProtocol
+
+  var schemeIcon = Constants.Bundle.SchemeIcon.blank {
```

Comment:
Could we move this property inside the viewModel and have it changed that way? This would allow us to test that:
> Given scheme Icon has been set, a callback gets received with an appropriate image